resources/views/partials/bootstrap-table.blade.php

```diff
@@ -197,7 +199,11 @@ function genericColumnObjLinkFormatter(destination) {
                     case 'deployed':
                         text_color = 'blue';
                         icon_style = 'fa-circle';
-                        text_help = '<label class="label label-default">{{ trans('general.deployed') }}</label>';
+                        if (!{{ $has_deployed_statuses }}) {
```

Comment:
Honestly, I don't love this, if just for the inconsistency. I'm willing to be overridden, but consistency is generally where we shine.